### PR TITLE
Replace _XXX with _* in constants

### PR DESCRIPTION
**Pull request by haszi**
Replace the replaceable parts of constants using `_XXX` with `_*` and add a `<replaceable>` tag. 

Also replaced some `<literal>` with `<constant>` tags and removed two `<link>`s as I'm planning on adding a feature to `PhD` that would link these `_*` constants to the first matching constant defined in the index.

### CODE REVIEW

**Review by nielsdos — APPROVED** [review]
Looks fine to me

**Review by Girgias — APPROVED** [review]

File: reference/image/functions/imagepng.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@
      <listitem>
       <para>
        Allows reducing the PNG file size. It is a bitmask field which may be
-       set to any combination of the <literal>PNG_FILTER_XXX</literal> 
+       set to any combination of the <constant>PNG_FILTER_<replaceable>*</replaceable></constant>
```
Comment (Girgias) [inline reference/image/functions/imagepng.xml:54]:
```suggestion
       set to any combination of the <constant>PNG_FILTER_<replaceable>*</replaceable></constant>
```

File: reference/image/functions/getimagesize.xml [hunk 2]
```diff
@@ -118,7 +118,7 @@
    </para>
   </note>
   <para>
-   Index 2 is one of the <link linkend="image.constants">IMAGETYPE_XXX constants</link> indicating 
+   Index 2 is one of the <constant>IMAGETYPE_<replaceable>*</replaceable></constant> constants indicating
```
Comment (Girgias) [inline reference/image/functions/getimagesize.xml:121]:
```suggestion
   Index 2 is one of the <constant>IMAGETYPE_<replaceable>*</replaceable></constant> constants indicating
```

**Comment by haszi** [comment]
Thanks for the whitespace fixes! 

Is there a way to have the CI automatically do this for us?

**Comment by Girgias** [comment]
> Thanks for the whitespace fixes!
> 
> Is there a way to have the CI automatically do this for us?

Ehhhh maybe, but I don't much about GitHub actions

**Pull request merged**